packages/block-editor/src/components/block-pattern-setup/style.scss

```diff
@@ -0,0 +1,124 @@
+.block-editor-block-pattern-setup {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: flex-start;
+	width: 100%;
+	border-radius: $radius-block-ui;
+	box-shadow: inset 0 0 0 $border-width $gray-900;
+	outline: 1px solid transparent; // Shown for Windows 10 High Contrast mode.
+
+	// TODO change to check parent.
+	&.view-mode-grid {
+		.block-editor-block-pattern-setup__toolbar {
+			justify-content: center;
+		}
+
+		.block-editor-block-pattern-setup__container {
+			display: grid;
+			grid-template-columns: 1fr 1fr;
+			grid-gap: $grid-unit-20;
+			padding: $grid-unit-20;
+			max-height: 550px;
+			overflow: auto;
+			margin: 0 $border-width $border-width $border-width;
+			width: calc(100% - #{ $border-width * 2 });
+			background: $white;
+
+			.block-editor-block-preview__container,
+			div[role="button"] {
+				cursor: pointer;
```

Comment:
What div has the 'button' role?

Also, Im not sure why but I am not seeing this pointer over the blocks being previewed - only their wrappers:

![cursor-pointer](https://user-images.githubusercontent.com/28742426/115568011-ddc7a500-a289-11eb-88d4-172548b917c8.gif)

Reply:
This hover interaction seems to work fine in the post editor, but the above gif is its behavior in the site editor.

Reply:
This has to do with the `BlockPreview` container to have the proper styles to make it feel that it's clickable/actionable.

>This hover interaction seems to work fine in the post editor, but the above gif is its behavior in the site editor.

This had to do with iframe and css-in-js: https://github.com/WordPress/gutenberg/issues/30553#issuecomment-824483510, that seems to be resolved 🎉